packages/client/src/lib/Sorting.re

```diff
@@ -0,0 +1,11 @@
+type sortable = Types.channel;
```

Comment:
I really don't like that this module is called ``sorting`` but it only uses ``Channel`` for some reason. Either figure out some nicer way of doing this (with a Functor, perhaps), or I wouldn't recommend it like this.

This also makes the assumption that the array has items with names that are strings.